src/data/press/coverage.yml

```diff
@@ -1,4 +1,9 @@
 -
+  title: "Decred Journal, July 2020"
+  url: "https://medium.com/decred/decred-journal-july-2020-7da45ed48660"
+  source: "medium.com/decred/journals/home"
+  date: "2020-08-07"
+  -
```

Comment:
This hyphen is indented incorrectly which is causing the build to fail

Reply:
Thanks @jholdstock, should be fixed now.